src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustCorporateValidatorTest.java

```diff
@@ -506,12 +508,147 @@ void testErrorRegisteredInCountryFormedInNotSupplied() {
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorIsStillInvolvedTrueAndCeasedDateNotNull() {
+        trustDataDtoList.get(0).getCorporates().get(0).setCorporateStillInvolvedInTrust(Boolean.TRUE);
+        trustDataDtoList.get(0).getCorporates().get(0).setCeasedDate(LocalDate.of(2020, 1, 1));
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(PARENT_FIELD, TrustCorporateDto.CEASED_DATE_FIELD);
+        String validationMessage = String.format(ValidationMessages.NULL_ERROR_MESSAGE, qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorIsStillInvolvedFalseAndCeasedDateNull() {
+        trustDataDtoList.get(0).getCorporates().get(0).setCorporateStillInvolvedInTrust(Boolean.FALSE);
+        trustDataDtoList.get(0).getCorporates().get(0).setCeasedDate(null);
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(PARENT_FIELD, TrustCorporateDto.CEASED_DATE_FIELD);
+        String validationMessage = String.format(ValidationMessages.NOT_NULL_ERROR_MESSAGE, qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorIsStillInvolvedFalseAndCeasedDateFuture() {
+        trustDataDtoList.get(0).getCorporates().get(0).setCorporateStillInvolvedInTrust(Boolean.FALSE);
+        trustDataDtoList.get(0).getCorporates().get(0).setCeasedDate(LocalDate.now().plusDays(1));
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(PARENT_FIELD, TrustCorporateDto.CEASED_DATE_FIELD);
+        String validationMessage = String.format(ValidationMessages.DATE_NOT_IN_PAST_ERROR_MESSAGE, qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorIsStillInvolvedFalseAndCeasedDateBeforeTrustCreationDate() {
+        trustDataDtoList.get(0).getCorporates().get(0).setCorporateStillInvolvedInTrust(Boolean.FALSE);
+        trustDataDto.setCreationDate(LocalDate.of(2020, 1, 1));
+        trustDataDtoList.get(0).getCorporates().get(0).setCeasedDate(LocalDate.of(1999, 12, 31));
+        Errors errors = trustCorporateValidator.validate(trustDataDtoList, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(PARENT_FIELD, TrustCorporateDto.CEASED_DATE_FIELD);
+        String validationMessage = String.format(ValidationMessages.CEASED_DATE_BEFORE_CREATION_DATE_ERROR_MESSAGE, qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
```

Comment:
Three of these empty lines can be removed.

Reply:
Updated this on github directly.